pkg/services/scm_provider/gitlab.go

```diff
@@ -60,6 +61,31 @@ func (g *GitlabProvider) GetBranches(ctx context.Context, repo *Repository) ([]*
 	return repos, nil
 }
 
+func (g *GitlabProvider) GetPullRequests(ctx context.Context, repo *Repository) ([]*Repository, error) {
+	repos := []*Repository{}
+	if !g.allPullRequests {
+		return repos, nil
+	}
+
+	pullRequests, err := g.listPullRequests(ctx, repo)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, pullRequest := range pullRequests {
+		repos = append(repos, &Repository{
+			Organization: repo.Organization,
+			Repository:   repo.Repository,
+			URL:          repo.URL,
+			Branch:       pullRequest.Title,
```

Comment:
This is my mistake. I will change it to `pullRequest.SourceBranch` which is the name of the source branch. This would be consistent with what the Github provider does as well.